src/migration/src/m20220915_153744_create_links_table.rs

```diff
@@ -47,14 +47,12 @@ impl MigrationTrait for Migration {
                     .col(
                         ColumnDef::new(Link::CreatedAt)
                             .timestamp_with_time_zone()
-                            .not_null()
-                            .extra(String::from("DEFAULT NOW()::timestamp")),
+                            .not_null(),
```

Comment:
In order to introduce this feature, migrations from current version must change. We must avoid custom SQL queries in order to keep SQLite support. I'm sure some features will not be supported by SQLite, but for current behavior
which is pretty lightweight there's no issues.

**Is never a good idea to change previous migrations, migrations should always be progressively changed and immutable**